k8s/analytics/templates/service.yaml

```
@@ -8,7 +8,8 @@ spec:
   type: {{ .Values.service.type }}
   ports:
     {{- range .Values.service.ports }}
-    - port: {{ .name }}
+    - name: {{ .name }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)